index.js

```diff
@@ -27,6 +27,20 @@ const role = {
   ROLE_REMOVE: ''
 }
 
+const ledger = {
+    DOMAIN: 1//, TODO: get the name an value of other sub ledgers

```

Comment:
I found this: `Pool=0; Domain=1; Config=2` from https://github.com/hyperledger/indy-node/blob/master/docs/source/requests.md#get_txn

Reply:
Thanks!